File: strategy/php/miniducksimulator.php
<?php
/**
 * file miniducksimulator.php
 *
 * @author kujaomega
 */
include('autoload.php');

$mallard_duck = new MallardDuck();
echo '<br>';
$mallard_duck->performFly();
echo '<br>';
$mallard_duck->performQuack();
echo '<br>Model duck: <br>';
$duck_model = new ModelDuck();
$duck_model->performFly();
$duck_model->setFlyBehavior( new FlyRocketPowered() );
echo '<br>';
$duck_model->performFly();

File: strategy/java/MiniDuckSimulator.java
/**
 * File MiniDuckSimulator.java
 * 
 * @author kujaomega
 */

/**
 * This is the MiniDuckSimulator that will run the main of the program.
 * 
 * class MiniDuckSimulator
 */
public class MiniDuckSimulator {
    
    /**
     * This is our main.
     * 
     * @param args 
     */
    public static void main (String [] args){
        Duck mallard=new MallardDuck();
        mallard.performQuack();
        mallard.performFly();
        Duck model=new ModelDuck();
        model.performFly();
        model.setFlyBehavior(new FlyRocketPowered());
        model.performFly();
    }
}
